[2022-02-02 17:56:54,612] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-02-02 17:56:54,635] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-02-02 17:56:54,637] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 17:56:54,638] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-02 17:56:54,640] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 17:56:54,666] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-01-01 00:00:00+00:00
[2022-02-02 17:56:54,675] {standard_task_runner.py:52} INFO - Started process 7797 to run task
[2022-02-02 17:56:54,686] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2019-01-01T00:00:00+00:00', '--job-id', '301', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmpfzt4b2b8', '--error-file', '/tmp/tmprtw7zxu8']
[2022-02-02 17:56:54,690] {standard_task_runner.py:77} INFO - Job 301: Subtask download_dataset_task
[2022-02-02 17:56:54,774] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-01-01T00:00:00+00:00 [running]> on host 700aeb16bae0
[2022-02-02 17:56:54,823] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-02 17:56:54,862] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-01T00:00:00+00:00
[2022-02-02 17:56:54,866] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-02 17:56:54,869] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-01.csv > /opt/***/yellow_tripdata_2019-01.csv']
[2022-02-02 17:56:54,901] {subprocess.py:85} INFO - Output:
[2022-02-02 18:00:39,343] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-02 18:00:39,414] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20190101T000000, start_date=20220202T175654, end_date=20220202T180039
[2022-02-02 18:00:39,499] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-02 18:00:39,543] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-10 12:43:50,515] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-02-10 12:43:50,528] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-02-10 12:43:50,530] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 12:43:50,531] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-10 12:43:50,532] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 12:43:50,554] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-01-01 00:00:00+00:00
[2022-02-10 12:43:50,569] {standard_task_runner.py:52} INFO - Started process 467 to run task
[2022-02-10 12:43:50,586] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2019-01-01T00:00:00+00:00', '--job-id', '660', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmp30kgnhpz', '--error-file', '/tmp/tmpc7mgt8nz']
[2022-02-10 12:43:50,591] {standard_task_runner.py:77} INFO - Job 660: Subtask download_dataset_task
[2022-02-10 12:43:50,694] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-01-01T00:00:00+00:00 [running]> on host e30f201fae61
[2022-02-10 12:43:50,754] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-10 12:43:50,801] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-01T00:00:00+00:00
[2022-02-10 12:43:50,803] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-10 12:43:50,805] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-01.csv > /opt/***/yellow_tripdata_2019-01.csv']
[2022-02-10 12:43:50,823] {subprocess.py:85} INFO - Output:
[2022-02-10 12:46:40,674] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-10 12:46:40,709] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20190101T000000, start_date=20220210T124350, end_date=20220210T124640
[2022-02-10 12:46:40,757] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-10 12:46:40,809] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-10 15:29:55,238] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-02-10 15:29:55,250] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-02-10 15:29:55,252] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 15:29:55,253] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-10 15:29:55,254] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 15:29:55,272] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-01-01 00:00:00+00:00
[2022-02-10 15:29:55,279] {standard_task_runner.py:52} INFO - Started process 115 to run task
[2022-02-10 15:29:55,291] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2019-01-01T00:00:00+00:00', '--job-id', '782', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmp6ire6mav', '--error-file', '/tmp/tmp33yssrz6']
[2022-02-10 15:29:55,293] {standard_task_runner.py:77} INFO - Job 782: Subtask download_dataset_task
[2022-02-10 15:29:55,406] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-01-01T00:00:00+00:00 [running]> on host 276a0ad90304
[2022-02-10 15:29:55,481] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-10 15:29:55,523] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-01T00:00:00+00:00
[2022-02-10 15:29:55,527] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-10 15:29:55,528] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-01.csv > /opt/***/yellow_tripdata_2019-01.csv']
[2022-02-10 15:29:55,567] {subprocess.py:85} INFO - Output:
[2022-02-10 15:30:54,238] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-10 15:30:54,434] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20190101T000000, start_date=20220210T152955, end_date=20220210T153054
[2022-02-10 15:30:54,557] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-10 15:30:54,723] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-10 16:30:04,898] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-02-10 16:30:04,924] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-02-10 16:30:04,926] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 16:30:04,927] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-10 16:30:04,928] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 16:30:04,951] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-01-01 00:00:00+00:00
[2022-02-10 16:30:04,965] {standard_task_runner.py:52} INFO - Started process 173 to run task
[2022-02-10 16:30:05,004] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2019-01-01T00:00:00+00:00', '--job-id', '927', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmpnuw3i02p', '--error-file', '/tmp/tmp5imft7sw']
[2022-02-10 16:30:05,007] {standard_task_runner.py:77} INFO - Job 927: Subtask download_dataset_task
[2022-02-10 16:30:05,172] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-01-01T00:00:00+00:00 [running]> on host 106a34111faf
[2022-02-10 16:30:05,238] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-10 16:30:05,286] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-01T00:00:00+00:00
[2022-02-10 16:30:05,288] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-10 16:30:05,289] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-01.csv > /opt/***/yellow_tripdata_2019-01.csv']
[2022-02-10 16:30:05,327] {subprocess.py:85} INFO - Output:
[2022-02-10 16:31:25,600] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-10 16:31:25,734] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20190101T000000, start_date=20220210T163004, end_date=20220210T163125
[2022-02-10 16:31:25,844] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-10 16:31:25,922] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
